tests/treeTest.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+from coalahtml.tree import Node, Tree
+from coalib.coala_main import run_coala
+from coalib.output.printers.ListLogPrinter import ListLogPrinter
+from coalahtml.helper import parse_file_dict
+
+
+class TreeTest(unittest.TestCase):
+
+    def setUp(self):
+        log_printer = ListLogPrinter()
+        file_dict = run_coala(log_printer=log_printer,
+                              autoapply=False)[2]
```

Comment:
Adding it by hand, will work only one system. It won't be system
independent. Right?
ᐧ

On Fri, May 27, 2016 at 2:42 PM, Attila <notifications@github.com> wrote:

> In tests/treeTest.py
> <https://github.com/coala-analyzer/coala-html/pull/46#discussion_r64877960>
> :
>
> > @@ -0,0 +1,36 @@
> > +import unittest
> > +
> > +from coalahtml.tree import Node, Tree
> > +from coalib.coala_main import run_coala
> > +from coalib.output.printers.ListLogPrinter import ListLogPrinter
> > +from coalahtml.helper import parse_file_dict
> > +
> > +
> > +class TreeTest(unittest.TestCase):
> > +
> > +    def setUp(self):
> > +        log_printer = ListLogPrinter()
> > +        file_dict = run_coala(log_printer=log_printer,
> > +                              autoapply=False)[2]
>
> Then use real one directory structure, but add it by hand. Tests should
> not be sophisticated.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-html/pull/46/files/404eb3d50f026d19d5a6738da9e8d0e1b5ffa05a..45b9d8e02c8e3c164f75093a3f89146b39bb2e61#r64877960>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHDgOadWFkexzcE1qJUgJqKCFUuvgz3mks5qFrVxgaJpZM4Ie05w>
> .
>